folium/folium.py

```diff
@@ -913,9 +917,21 @@ def json_style(style_cnt, line_color, line_weight, line_opacity,
             # Create legend.
             name = legend_name or columns[1]
             leg_templ = self.env.get_template('d3_map_legend.js')
-            legend = leg_templ.render({'lin_max': int(domain[-1]*1.1),
-                                       'tick_labels': tick_labels,
-                                       'caption': name})
+
+            if freescale==False:
+                legend = leg_templ.render({'lin_min': 0,
+                           'lin_max': int(domain[-1]*1.1),
+                           'tick_labels': tick_labels,
+                           'caption': name})
+
+
+            else:
+                legend = leg_templ.render({'lin_min':  domain[0],
+                                           'lin_max':  domain[-1],
```

Comment:
OK.  `'lin_min': int(domain[0]*1.1)` was probably a copy-and-paste mistake from my original PR).

The idea was to extend the legend a little bit.  We could be smart about it and use a sign function, expand, and re-added the sign.   But the legends look good without this, so lets leave it be.